packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/picture-element.twig

```diff
@@ -1,11 +1,24 @@
-<picture class="ma__picture">
-  {% if picture.srcSets %}
-    {% for src in picture.srcSets %}
-    <source srcset="{{ src.url }}" media="(min-width: {{ src.width }}px)">
-    {% endfor %}
-  {% endif %}
+{% if picture.srcSets or picture.defaultImage %}
+  <picture class="ma__picture">
+    {% if picture.srcSets %}
+      {% for src in picture.srcSets %}
+        {% set media = "" %}
+        {% set source = src.url %}
 
-  {% if picture.defaultImage %}
-    <img src="{{ picture.defaultImage }}">
-  {% endif %}
-</picture>
+        {% if src.width is not empty %}
+          {% set media = '(min-width: ' ~ src.width ~ 'px)' %}
```

Comment:
```suggestion
          {% set media = '(max-width: ' ~ src.width ~ 'px)' %}
```

@Javi-er shouldn't this be updated in mayflower? 
Can you update the data in mayflower to reflect the openmass breakpoints?

Reply:
Just changed this to max-width instead. Also updated the mayflower component to match as close as possible what we have in openmass.